docs/docs/usage/api.md

```diff
@@ -8,12 +8,13 @@
 
     **Parameters:**
 
-    | Name          | Description                                                                                | Default                |
-    | ------------- | ------------------------------------------------------------------------------------------ | ---------------------- |
-    | `installer`   | Which `conda` installer to download, either `miniconda` or `miniforge`                     | `miniconda`            |
-    | `version`     | Version of `conda` to download                                                             | `4.10.3`               |
-    | `quiet`       | `True` if `conda` output should be hidden                                                  | `True`                 |
-    | `timeout`     | How many seconds each execute action can take                                              | `3600`                 |
+    | Name            | Description                                                                                | Default                |
+    | --------------- | ------------------------------------------------------------------------------------------ | ---------------------- |
+    | `installer`     | Which `conda` installer to download, either `miniconda` or `miniforge`                     | `miniconda`            |
+    | `install_mamba` | Whether to install `mamba`, which is a faster drop-in replacement for `conda`              | `miniconda`            |
+    | `version`       | Version of `conda` to download                                                             | `4.10.3`               |
```

Comment:
I think it's called like this now:

```suggestion
    | `conda_version` | Version of `conda` to download                                                             | `4.10.3`               |
```

Reply:
Oh good catch, I forgot to add `mamba_version` doc too.

Reply:
And the default value of `install_mamba` is listed as `miniconda` 😛